src/main/java/com/redhat/che/workspace/services/telemetry/woopra/AnalyticsManager.java

```diff
@@ -318,6 +320,15 @@ void onActivity() {
       lastActivityTime = System.currentTimeMillis();
     }
 
+    private String makePluginString(List<String> pluginNames) {
```

Comment:
Can we try to use some java 8 magic here smth. like `stream().collect(Collectors.joining(", "));` ?